cli/sourcemaps.ts

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
+import { Command, Option, OptionValues } from 'commander';
+import figlet from 'figlet';
+import fs from 'fs';
+
+const program: Command = new Command();
+
+console.log(figlet.textSync('Instabug React Native CLI'));
```

Comment:
**suggestion:** I think "Instabug CLI" is less verbose and matches the usage of `yarn instabug upload-sourcemaps`. What do you think?

```suggestion
console.log(figlet.textSync('Instabug CLI'));
```

Reply:
It depends on the command usage as you mentioned, which we'll discuss.